compile/api/src/main/scala/sbt/ClassToAPI.scala

```diff
@@ -127,14 +133,54 @@ object ClassToAPI {
   def upperBounds(ts: Array[Type]): api.Type =
     new api.Structure(lzy(types(ts)), lzyEmptyDefArray, lzyEmptyDefArray)
 
-  def fieldToDef(enclPkg: Option[String])(f: Field): api.FieldLike =
```

Comment:
This would be a binary compatibility breakage.  May be ok, but would prefer to see this deprecated and an overload/diferent name instead.